parla/task_runtime.py

```diff
@@ -638,6 +637,8 @@ def __init__(self, environments: Collection[TaskEnvironment], n_threads: int = N
         self._unassigned_resources = ResourcePool(multiplier=max_worker_queue_depth*1.0)
         self._worker_threads = [WorkerThread(self, i) for i in range(n_threads)]
         self._should_run = True
```

Comment:
_should_run is marked True during `super().__init__()` (of ControllableThread) so this remarking can also be removed?

Reply:
Good catch. Yes. Go ahead and do that in this PR if you like.